### PR TITLE
Fix(VInput): Browser Autocomplete

### LINKED ISSUE #9516: [Bug Report] input field in v-autocomplete should be autocomplete="off" instead of disabled
### Environment
**Vuetify Version:** 2.1.7
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.70
**OS:** Linux x86_64

### Steps to reproduce
Write many many times e.g. "Comboboxes" and after a while Google Chrome gives you a second "dropdown" with "Comboboxes" this is because the input field is like this:

```
<input id="input-4" type="text" autocomplete="disabled">
```

but should be rather like this:

```
<input id="input-4" type="text" autocomplete="off">
```

### Expected Behavior
no extra dropdown from Chrome

### Actual Behavior
second overlaying dropdown from Chrome

### Reproduction Link
<a href="https://codepen.io/samuelba/pen/eYYGGeN" target="_blank">https://codepen.io/samuelba/pen/eYYGGeN</a>

### PR DESCRIPTION
**Pull request by melbahja**
## Description
Disable browser autofill menu on VSelect, VCombobox, VAutocomplete

## Motivation and Context
fixes #9516

## How Has This Been Tested?
unit | visually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/lang/en/components/Selects.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "heading": "# Selects",
   "headingText": "Select fields components are used for collecting user provided information from a list of options.",
   "objectAlert": "When using objects for the **items** prop, you must associate **item-text** and **item-value** with existing properties on your objects. These values are defaulted to **text** and **value** and can be changed.",
+  "autocompleteAlert": "Browser autocomplete are set to off by default, and may vary given that browsers such as chrome may ignore it. [MDN](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion)",
```
Comment (johnleider) [inline packages/docs/src/lang/en/components/Selects.json]:
```suggestion
  "autocompleteAlert": "Browser autocomplete is set to off by default and may vary by browser and may be ignored. [MDN](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion)",
```

**Comment by lilalinux** [comment]
At least Chrome takes the attribute autocomplete literally, so "off" is no better than any other fixed value. The recommended fix (according to google) is to use random values (or semantic random values):
https://gist.github.com/niksumeiko/360164708c3b326bd1c8#gistcomment-2956701

**Comment by melbahja** [comment]
@lilalinux
> The problem is, if the user comes back to fill this form again, my-special-phonenumber-field isn't a new thing anymore. Chrome autocompletes. In my case, it does it wrong and modifies some fields it should not. which is boring as people validate a form with modified values and they don't notice....

The problem here is you need a unique value each and every time. and how about other browsers?
I think maybe if we can use selects without `input` HTML tag like MD web:
https://material.io/develop/web/components/input-controls/select-menus/

**Comment by lilalinux** [comment]
I *think* random values would work for other browsers, too. I don't see a problem in creating a random value each time the input is used, why not setting a UUID via JS?

An alternative approach of the linked discussion seems to be dynamically setting the input readonly.

**Comment by melbahja** [comment]
Yes I think `readonly` with `autocomplete="off"` should works as expected.

File: packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts [hunk 2]
```diff
@@ -283,7 +283,8 @@ export default VSelect.extend({
 
       input.data = input.data || {}
       input.data.attrs = input.data.attrs || {}
-      input.data.attrs.autocomplete = input.data.attrs.autocomplete || 'disabled'
+      input.data.attrs.readonly = true
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts]: That makes autocomplete and combobox readonly

Comment (melbahja) [reply packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts]: I didn't think of that :man_facepalming:

**Comment by johnleider** [comment]
Requesting final comments @vuetifyjs/core-contributors @vuetifyjs/core-team

**Pull request merged**